testament/tester.nim

```diff
@@ -90,6 +90,8 @@ proc execCmdEx2(command: string, args: openarray[string], options: set[ProcessOp
                 output: TaintedString,
                 exitCode: int] {.tags:
                 [ExecIOEffect, ReadIOEffect, RootEffect], gcsafe.} =
+  echo (msg: "execCmdEx2", command: command & " " & args.quoteShellCommand,
```

Comment:
Well you can ``echo`` this on failure only, right?

Reply:
done